test/server/getters/get-profit-loss.js

```diff
@@ -2,11 +2,20 @@ const Augur = require("augur.js");
 const assert = require("chai").assert;
 const setupTestDb = require("../../test.database");
 const {calculateBucketProfitLoss, getProfitLoss, bucketRangeByInterval} = require("../../../build/server/getters/get-profit-loss");
+const sqlite3 = require("sqlite3");
+const Knex = require("knex");
+const { postProcessDatabaseResults } = require("../../../build/server/post-process-database-results");
 
 const START_TIME = 1506474500;
 const MINUTE_SECONDS = 60;
 const HOUR_SECONDS = MINUTE_SECONDS*60;
 
+const BigNumber = require('bignumber.js');
+BigNumber.config({
```

Comment:
Should we also call `BigNumber.config` in the source files?  I know this ended up not being an issue earlier, but it would be good to make sure `BigNumber` here and in augur.js are configured the same way.

Reply:
You're right we should move this, I'll update.